opacus/grad_sample/grad_sample_module.py

```diff
@@ -296,6 +303,9 @@ def capture_activations_hook(
             module.activations = []
         module.activations.append(forward_input[0].detach())  # pyre-ignore
 
+        for p in module.parameters():
```

Comment:
Do you think it will help to add a comment above this line mentioning this is for tied parameters? A link to the PR itself is even better. Your PR summary is stellar!